src/error.rs

```diff
@@ -11,6 +11,8 @@ pub enum DbErr {
     RecordNotFound(String),
     /// A custom error
     Custom(String),
+    /// Error occurred while parsing value into [ActiveEnum](crate::ActiveEnum)
```

Comment:
Just a slight change, I think it is a more general error, which can be documented as 'Error occurred while parsing value into target type'